backend/utils.py

```diff
@@ -16,6 +17,12 @@
 from backend.config import CONFIG
 
 
+def powerset(iterable):
```

Comment:
`powerset`: I don't see where this is used at the moment; fill me in if you want.
I like `chain`; don't use it that often. Haven't used `combinations` before.